### PR TITLE
Migrate Farmlands External Asset Batch Page

### PR DESCRIPTION
**Pull request by AndyClifford**
**Ticket:** https://www.notion.so/centrapay/Lanterns-d784e8a5f9fa44328a5d0ae1fd85d37e?p=c6d64de29a2245358a5e9b7eed5edbcc&pm=s

### CODE REVIEW

**Comment by MeganSteenkamp** [comment]
@AndyClifford am I correct in thinking this is hidden in our usual nav? How will we deal with this?

File: src/content/api/batch-types/farmlands-external-asset.mdx [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+
+  <Property name="primary" type="boolean">
+    `true` if the Contact is the owner of the Farmlands Account.
+  </Property>
+
+  <Property name="cards" type="array">
+    [Card](#card-model) list used for payment for the authenticated subject. When empty all Cards are removed.
+  </Property>
+</Properties>
+
```
Comment (MeganSteenkamp) [inline src/content/api/batch-types/farmlands-external-asset.mdx:77]:
```suggestion
---

```

File: src/content/api/batch-types/farmlands-external-asset.mdx [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+  <Property name="expiry" type="timestamp">
+    Payments will be accepted until this time.
+  </Property>
+
+  <Property name="farmlandsStatus" type="string">
+    Private field used in all Farmlands payment transaction notifications.
+  </Property>
+</Properties>
+
```
Comment (MeganSteenkamp) [inline src/content/api/batch-types/farmlands-external-asset.mdx:104]:
```suggestion

---
```

**Comment by AndyClifford** [comment]
> @AndyClifford am I correct in thinking this is hidden in our usual nav? How will we deal with this?

Yeah that's right, was just gonna deal with it later but we can specify 'Do Not Include' as the nav path and I think it just won't show up

**Review by MeganSteenkamp — APPROVED** [review]

**Pull request merged**